nodejs-JWT/app.js

```diff
@@ -28,23 +30,26 @@ env.addFilter("indication_option", function(name) {
 // Open the config file
 let config = openConfig("../" + config_file);
 
-// Check and set valid region
-validateRegion(config);
-
 // Set variables from config
 let port = config["port"]; // the port the server will be hosted on
+let secretKey = config["secret_key"] // session secret key
 let DRUGBANK_API = "https://api.drugbankplus.com/v1/"; // the DrugBank API link
-let DRUGBANK_REGION = config["region"]; // the region to get results from
-let DRUGBANK_API_KEY = config["auth-key"]; // DrugBank API key    
 
-// DrugBank headers that need to be set. Needs to be sent with any request to the 
-// DrugBank API as it contains the API authorization key, which validates access.
-let DRUGBANK_HEADERS = 
-    {
-        "Authorization": DRUGBANK_API_KEY,
-        "Content-Type": "application/json",
-        "Accept": "application/json"
-    };    
+// app.set('trust proxy', 1) // trust proxy uncomment if running behind a proxy like nginx
+
+// Setup server side sessions.
+app.use(session({
+    secret: secretKey,
+    name: 'drugbank-jwt-sample',
+    resave: true,
+    cookie: {
+        httpOnly: true,
+        secure: false, // secure false when running locally or without https
```

Comment:
maybe this could be read from the config as well? I think it would be nicer for deployment that we could just set some config variables and not need to change the code at all.

I think the same would make sense for the trust proxy setting above.

Reply:
Ya makes sense!